lib/triangles/is-triangle.js

```diff
@@ -0,0 +1,21 @@
+function isRightTriangle (side1, side2, side3) {
+  if (side1 <= 0 || side2 <= 0 || side3 <= 0){
+        return false;
+    } else if ((Math.pow(side1, 2) + Math.pow(side2, 2)) === Math.pow(side3, 2)) {
```

Comment:
you've got some repeated logic here. it doesn't matter what order you add the first two numbers together, you don't need to do it twice 👍